src/Windowing/Silk.NET.SDL/SDL.cs

```diff
@@ -1,21 +1,918 @@
 using System;
-using Silk.NET.Core.Loader;
-using Silk.NET.Core.Native;
-using Silk.NET.Core.Attributes;
 using Ultz.SuperInvoke;
 
 #pragma warning disable 1591
 
 namespace Silk.NET.SDL
 {
-    public partial class SDL
+    public partial class Sdl
     {
-        public static SDL GetApi()
+        public const uint InitTimer = 0x00000001;
+        public const uint InitAudio = 0x00000010;
+        public const uint InitVideo = 0x00000020;
+        public const uint InitJoystick = 0x00000200;
+        public const uint InitHaptic = 0x00001000;
+        public const uint InitGamecontroller = 0x00002000;
+        public const uint InitEvents = 0x00004000;
+        public const uint InitSensor = 0x00008000;
+        public const uint InitNoparachute = 0x00100000;
+
+        public const uint InitEverything = InitTimer | InitAudio | InitVideo |
+                                           InitEvents | InitJoystick | InitHaptic |
+                                           InitGamecontroller | InitSensor;
+
+        public const string HintFramebufferAcceleration =
+            "SDL_FRAMEBUFFER_ACCELERATION";
+
+        public const string HintRenderDriver =
+            "SDL_RENDER_DRIVER";
+
+        public const string HintRenderOpenglShaders =
+            "SDL_RENDER_OPENGL_SHADERS";
+
+        public const string HintRenderDirect3DThreadsafe =
+            "SDL_RENDER_DIRECT3D_THREADSAFE";
+
+        public const string HintRenderVsync =
+            "SDL_RENDER_VSYNC";
+
+        public const string HintVideoX11Xvidmode =
+            "SDL_VIDEO_X11_XVIDMODE";
+
+        public const string HintVideoX11Xinerama =
+            "SDL_VIDEO_X11_XINERAMA";
+
+        public const string HintVideoX11Xrandr =
+            "SDL_VIDEO_X11_XRANDR";
+
+        public const string HintGrabKeyboard =
+            "SDL_GRAB_KEYBOARD";
+
+        public const string HintVideoMinimizeOnFocusLoss =
+            "SDL_VIDEO_MINIMIZE_ON_FOCUS_LOSS";
+
+        public const string HintIdleTimerDisabled =
+            "SDL_IOS_IDLE_TIMER_DISABLED";
+
+        public const string HintOrientations =
+            "SDL_IOS_ORIENTATIONS";
+
+        public const string HintXinputEnabled =
+            "SDL_XINPUT_ENABLED";
+
+        public const string HintGamecontrollerconfig =
+            "SDL_GAMECONTROLLERCONFIG";
+
+        public const string HintJoystickAllowBackgroundEvents =
+            "SDL_JOYSTICK_ALLOW_BACKGROUND_EVENTS";
+
+        public const string HintAllowTopmost =
+            "SDL_ALLOW_TOPMOST";
+
+        public const string HintTimerResolution =
+            "SDL_TIMER_RESOLUTION";
+
+        public const string HintRenderScaleQuality =
+            "SDL_RENDER_SCALE_QUALITY";
+
+        /* Only available in SDL 2.0.1 or higher. */
+        public const string HintVideoHighdpiDisabled =
+            "SDL_VIDEO_HIGHDPI_DISABLED";
+
+        /* Only available in SDL 2.0.2 or higher. */
```

Comment:
Unsure right now whether this is generated or now, but maybe these should be `/// <remarks>`

Reply:
Fixed.